src/api/attributes.spec.ts

```
@@ -783,6 +783,48 @@ describe('$(...)', () => {
+    it('(KEY) : should remove regardless of case in HTML documents', () => {
+      const $fruits = $('#fruits');
+      expect($fruits.attr('id')).not.toBeUndefined();
+      $fruits.removeAttr('ID');
+      expect($fruits.attr('id')).toBeUndefined();
+    });
+
+    it('(key) : should keep SVG attribute names case-sensitive', () => {
+      const $svg = load('<svg viewBox="0 0 10 10"></svg>');
+      expect($svg('svg').attr('viewBox')).toBe('0 0 10 10');
+      // SVG attribute names are case-sensitive, so a lowercase lookup
+      // must not remove the case-preserved attribute.
```

ESLint

[error] Expected a block comment instead of consecutive line comments.

(multiline-comment-style)